### ISSUE TITLE
Count index missing a node after live import

### ISSUE THREAD

**Issue by mehdiym** [dgraph-io/dgraph#3528]
- What version of Dgraph are you using?
1.0.15

- What is the hardware spec (RAM, OS)?
16Go RAM, Linux Debian stable

- Steps to reproduce the issue (command/config used to run Dgraph).
  * set a schema including a @count index (and a @reverse index) on a predicate
  * live load a previous export
  * query with a count function at root : eq(count(predicate), 1)

- Expected behaviour and actual result.
Expected : No count different than 1
Actual : One node with a count greater than one

My request in ratel:
```
{
  me(func: eq(count(answer), 1)) {
    count(answer)
  }
}
```

The response:
```
{
  "extensions": {
    "server_latency": {
      "parsing_ns": 18314,
      "processing_ns": 2869208,
      "encoding_ns": 1473700
    },
    "txn": {
      "start_ts": 1163
    }
  },
  "data": {
    "me": [
      {
        "count(answer)": 2
      },
      {
        "count(answer)": 1
      },
      {
        "count(answer)": 1
      },
      {
        "count(answer)": 1
      }
    ]
  }
}
```

If I go to the schema tab, disable the @count index and re-enable it, then the query works as expected.

Note: My live import inserts many nodes but it is always that same node that gets wrong.

**Comment by MichelDiz** [comment]
Hi, Can you share a RDF example that we can reproduce this?

**Comment by danielmai** [comment]
Interesting that re-indexing the `@count` index makes the query work as expected.

I can reproduce the issue the steps you shared. Funny, when I run `dgraph live --conc 1` to turn off concurrent mutations (default concurrency is 10), the query works as expected. Even `--conc 2` shows the query issue. I suspect this issue is related with concurrent mutations to the same subject-predicate.

**Comment by mehdiym** [comment]
I'm "glad" you can reproduce it.
Please note that I just reproduced the same bug but on nodes that were NOT imported by the live tool: on a fully generated database.
It happens during running my high concurrency integration tests.

**Comment by danielmai** [comment]
This issue is fixed with #3545 that has been merged in master.
